### ISSUE TITLE
Adding the "50/50 component" to landing + default templates

### ISSUE THREAD

**Issue by koalie** [w3c/w3c-website#94]
As we feed content or migrate pages to the CMS, we realize that the 50/50 component would be mightily helpful in the landing and default (basic) templates.

We understand the design of this component aims to give equivalent space and afford mutual support to imagery and text for a strong call to action, but the truth is that even without call to action buttons and/or links, the 50/50 component is without doubt the best design for our usage because we are more text oriented and recognize that some imagery is helpful (while the "image" component gives full visibility to imagery which we currently lack and we know will be hard to come by although we will endeavour to include more imagery.)

Use-cases we've already identified:
* Press Release (uses the "default" template) https://www-dev.w3.org/press-releases/2021/webrtc-rec/ (staged version) features a very prominent image while if you look at the original version https://www.w3.org/2021/01/pressrelease-webrtc-rec.html.en it isn't as important to be displayed so big.
* Any homepage news (e.g.,  https://www.w3.org/blog/news/archives/9373 which shows a thumbnail of an image)
* Any W3C Blog post (e.g., https://www.w3.org/blog/2021/12/idpd/ which shows a larger illustration)

**Comment by vivienlacourba** [comment]
cc: @JuliaSang to review assignment of this ticket

**Comment by gosko** [comment]
A couple more examples of blog posts where the images would not work well if scaled up to full-width (and possibly not even half-width):

- [CSS X](https://www.w3.org/blog/2020/03/css-x/) (see also: [test import on www-dev that demonstrates the issue](https://www-dev.w3.org/blog/2020/css-x/))
- [WCAG 3 FPWD Published](https://www.w3.org/blog/2021/01/wcag-3-fpwd/)

**Comment by maries24** [comment]
The 50/50 component is now available in the CMS for default pages, blog posts, press releases, and news.
@jean-gui are you happy to take care of pulling the data in the front-end app?

**Comment by JuliaSang** [comment]
Hi @vivienlacourba @koalie 

I just realised this was added to the high priority list end of last week. As we didn't spot the addition, Marie has spent time working on it. However we'd recommend against using the 50/50 component for blog posts, press releases, and news - as the 50/50 was designed for pages without a sidebar (i.e. full width pages) and won't display content as well on pages with a sidebar. 

From the issue described above, we would hope the fix for https://github.com/w3c/w3c-website/issues/98 would give you enough flexibility to solve the use cases you have mentioned.
